### PR TITLE
fix: Fix incorrect privilegekeyadd command

### PR DESCRIPTION
**Pull request by HoneyBBQ**
[New Pull Request](https://github.com/Unity-Technologies/go-ts3/pull/45)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Features
 Installation
 ------------
 ```sh
-go get -u github.com/multiplay/go-ts3
+go get -u github.com/honeybbq/go-ts3
```
Comment (stevenh) [inline README.md:12]: Bug: You’ve changed the name of the module which isn’t needed

File: README.md [hunk 2]
```diff
@@ -44,10 +44,6 @@ func main() {
 }
 ```
 
-Documentation
```
Comment (stevenh) [inline README.md:47]: suggestion: this should be updated to point to thr new docs site instead of removed

File: server_cmds.go [hunk 3]
```diff
@@ -461,3 +461,16 @@ func (s *ServerMethods) ClientDBList() ([]*DBClient, error) {
 	}
 	return dbclients, nil
 }
+
+type Snapshot struct {
```
Comment (stevenh) [inline server_cmds.go:465]: Question: this looks unrelated to the described change

File: server_cmds_test.go [hunk 4]
```diff
@@ -402,6 +402,18 @@ func testCmdsServer(t *testing.T, c *Client) {
 		assert.Equal(t, expected, clients)
 	}
 
+	serversnapshotcreate := func(t *testing.T) {
```
Comment (stevenh) [inline server_cmds_test.go:405]: As above unrelated changes

File: server_cmds.go [hunk 5]
```diff
@@ -368,7 +368,7 @@ func (s *ServerMethods) PrivilegeKeyAdd(ttype, id1, id2 int, options ...CmdArg)
 		Token string
 	}{}
 	options = append(options, NewArg("tokentype", ttype), NewArg("tokenid1", id1), NewArg("tokenid2", id2))
-	_, err := s.ExecCmd(NewCmd("privilegekeylist").WithArgs(options...).WithResponse(&t))
+	_, err := s.ExecCmd(NewCmd("privilegekeyadd").WithArgs(options...).WithResponse(&t))
```
Comment (stevenh) [inline server_cmds.go:371]: This looks like the only needed change

**Review by stevenh — CHANGES_REQUESTED** [review]
Looks like a bunch of unrelated changes have got bundled in here

File: server_cmds.go (hunk 5 above)
Comment (HoneyBBQ) [reply server_cmds.go:371]: I forgot to close the pull request. Since your repository hasn’t been updated for a long time, I forked it and made the changes myself.

Comment (HoneyBBQ) [reply server_cmds.go:371]: > This looks like the only needed change

I forgot that the pull request here is based on the main branch after I forked it. Sorry about that.

Comment (HoneyBBQ) [reply server_cmds.go:371]: > This looks like the only needed change

I have opened a new pull request, please review it.

**Comment by HoneyBBQ** [comment]
I have opened a new pull request, please review it.
